src/Middleware/Localization/src/RequestLocalizationMiddleware.cs

```diff
@@ -146,6 +147,11 @@ public async Task Invoke(HttpContext context)
 
             SetCurrentThreadCulture(requestCulture);
 
+            if (_options.SetContentLanguageResponseHeader)
+            {
+                context.Response.Headers.Add(HeaderNames.ContentLanguage, requestCulture.UICulture.Name);
```

Comment:
@pranavkm `AddXXX` is make sense while we adding the header by enabling this property